### ISSUE TITLE
why NMS op coordinates using y1,x1,y2,x2?

### ISSUE THREAD

**Issue by jinfagang** [onnx/onnx#2697]
the doc says:

```
center_point_box : int (default is 0)
Integer indicate the format of the box data. The default is 0. 0 - the box data is supplied as [y1, x1, y2, x2] where (y1, x1) and (y2, x2) are the coordinates of any diagonal pair of box corners and the coordinates can be provided as normalized (i.e., lying in the interval [0, 1]) or absolute. Mostly used for TF models. 1 - the box data is supplied as [x_center, y_center, width, height]. Mostly used for Pytorch models.
```

Why using y1,x1,y2,x2 as default rather than x1,y1,x2,y2?

**Comment by linkerzhang** [comment]
@HectorSVC

**Comment by HectorSVC** [comment]
even using x1,y1,x2,y2, there could be people ask why not y1,x1,y2,x2.
The major reason is because NMS is mostly used in Tensorflow, so we follow the tensorflow way which also make the model converter easier.

**Comment by jinfagang** [comment]
Resonable, however the coordinates used in nms should be more natrual as it used more widely in OpenCV style. Also I think y1,x1,y2,x2 should consistent with x, y, w, h which in x->y order.

**Comment by RuRo** [comment]
Also, if `center_point_box` is true, then its `x, y, w, h` and not `y, x, h, w`. I mean, I don't think, it actually matters, since NMS is supposed to be invariant towards coordinate transposition, but this is definitely a little bit weird and counterintuitive.

**Comment by jinfagang** [comment]
both can work, but sometimes need swap axis to fix standard which add compute time.

**Comment by HectorSVC** [comment]
center_point_box = true  is for Pytorch. Tensorflow & Pytorch are the framework we referenced. Please add Transpose op where it requires.
